internal/view/summary.go

```diff
@@ -93,6 +103,14 @@ var summaryTemplate *template.Template = template.Must(template.New("summary").P
 
   Open issues:     {{.OpenIssues}}`))
 
+var summaryWithUntestedTemplate *template.Template = template.Must(template.New("summary").Parse(`{{.Title}}
```

Comment:
I'm not that familiar with the `html/template` library, but can't we have some kind of conditional logic in the original template instead of creating an almost identical new one?

Reply:
I tried it out, but it was not so usable as I wanted. What I tried was to add `{{if }}{{- end}}` but this was easier to read. I might experiment with it later.